File: Semester 4/Course Assignment/Java code/models/PartDestination.java
package models;

public class PartDestination {
	private String partType;
	private String receiverName;
	private String receiverAddress;
	private String receiverCountry;
	
	public PartDestination(String partType, String receiverName, String receiverAddress, String receiverCountry) {
		this.partType = partType;
		this.receiverName = receiverName;
		this.receiverAddress = receiverAddress;
		this.receiverCountry = receiverCountry;
	}

	public String getPartType() {
		return partType;
	}

	public String getReceiverName() {
		return receiverName;
	}

	public String getReceiverAddress() {
		return receiverAddress;
	}

	public String getReceiverCountry() {
		return receiverCountry;
	}
	
	public String toString()
	{
		return partType + " " + receiverName + " " + receiverAddress + " " + receiverCountry;
	}
}

File: Semester 4/Course Assignment/C# code/FacilityClient/ClassLibrary/PartDestination.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClassLibrary
{
    public class PartDestination
    {
        public PartDestination(string partType, string receiverName, string receiverAddress, string receiverCountry)
        {
            this.partType = partType;
            this.receiverName = receiverName;
            this.receiverAddress = receiverAddress;
            this.receiverCountry = receiverCountry;
        }
        
        public string partType { get; set; }
        public string receiverName { get; set; }
        public string receiverAddress { get; set; }
        public string receiverCountry { get; set; }
    }
}
